ui/raidboss/data/03-hw/raid/a8s.js

```diff
@@ -561,4 +561,131 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'cn',
+      'replaceSync': {
+        'Vortexer': '环旋者',
+        'Swindler': '欺诈者',
+        'Steam Regulator B': '蒸汽调整者β',
+        'Steam Regulator A': '蒸汽调整者α',
+        'Steam Gasket': '蒸汽垫',
+        'Power Plasma Gamma': '强离子体γ',
+        'Power Plasma Beta': '强离子体β',
+        'Onslaughter': '突击者',
+        'J Storm': '正义旋风',
+        'Hidden Mine': '隐形地雷',
+        'Brute Justice': '残暴正义号',
+        '(?! )Brawler(?<! )': '争斗者',
+        'Blaster Mirage': '爆破者幻象',
+        '(?<! )Blaster(?! )': '爆破者',
+        'Eye Of The Chakram': '战轮光束',
+        'Steam Chakram': '蒸汽战轮',
+        'Reset-': '重置',
+        'Added new combatant Blaster': '增加新的爆破者',
```

Comment:
Does the Chinese version of ACT translate the "added new combatant" text? I didn't think it did.